### ISSUE TITLE
Add an output flag

### ISSUE THREAD

**Issue by ArsenArsen** [phw/peek#53]
Hello! I wanted to use peek in my image uploader, so I was wondering is there a possibility for something like `peek -o OUTPUT.gif` that will prompt to record a single gif and save it to `OUTPUT.gif` Thanks!

**Comment by seanmavley** [comment]
If terminal-only commands will be added, then I think recording time interval and perhaps dimensions should be specified too.

So something like:

`peek -o OUTPUT.gif -t 10 -w 250 -h 250`

where `-o` is output filename, `-t` is seconds to record, `-w` and `-h` is width and height in pixels respectively.

Or something along the lines of these

**Comment by ArsenArsen** [comment]
Seems good, but what I was thinking was still popping up the record screen and have it Dave by a terminal command. I still want it to be a user chosen area. 

Great work by the way.

**Comment by phw** [comment]
Yes, specifying the file output could make sense. Will see if I can add it.

@seanmavley Not sure about that. I understand the idea here, but actually there are tools just for that. E.g. you can achieve scriptng like that with ffcast + ffmpeg. Does not really fit well into Peek, which really is supposed to be a GUI.
